src/include/platform/OTARequestorStorage.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
```

Comment:
platform should not depend on app/

Reply:
This is not `app`. It's `app-common`.

Reply:
It's still generated code that doesn't belong in platform

Reply:
Why not?  This is code that is the reification of the spec into code form.  Is the argument that "platform" just should not depend on data model concepts at all?  What is the difference between data model types that we put into `core` (like NodeId) and data model types that are in cluster-objects (like enum definitions)?

Reply:
Ok, I'm not sure what's the outcome, but I moved the interface to `src/app/clusters/ota-requestor` to move forward with the PR.